environment.yml

```diff
@@ -1,27 +1,30 @@
 name: snorkeling
 dependencies:
-- beautifulsoup4=4.5.1
+- beautifulsoup4=4.6.0
 - enum34=1.1.6
-- futures=3.0.3
-- ipywidgets=5.2.2
+- futures=3.1.1
+- ipywidgets=6.0.0
 - jupyter=1.0.0
-- jupyter_client=4.4.0 
-- jupyter_console=5.0.0 
-- jupyter_core=4.2.0
-- llvmlite=0.13.0
-- lxml=3.7.2
-- matplotlib=1.5.3
-- nltk=3.2.1
-- numba=0.28.1
-- numpy=1.11.1 
-- pandas=0.18.1 
-- requests=2.11.1 
+- jupyter_client=5.0.1
+- jupyter_console=5.1.0
+- jupyter_core=4.3.0
+- llvmlite=0.18.0
+- lxml=3.6.4
+- matplotlib=2.0.2
+- nltk=3.2.4
+- numba=0.33.0
+- numpy=1.12.1 
+- pandas=0.20.2 
+- psycopg2==2.7.1
+- requests=2.14.2
 - seaborn=0.7.1
-- scikit-image=0.12.3
-- scikit-learn=0.17.1
-- scipy=0.18.1  
-- sqlalchemy=1.1.4
+- scikit-image=0.13.0
+- scikit-learn=0.18.1
+- scipy=0.19.0 
+- sqlalchemy=1.1.11
 - pip:
     - git+https://github.com/HazyResearch/numbskull@40ac1af20538c17b4726c963c69adcb81314efa5
-    - neo4j-driver==1.1.0
-    - tensorflow==0.12.1
+    - neo4j-driver==1.3.1
+    - tensorflow==1.2.0
+    - tika==1.14.1
```

Comment:
Presumably since v1.2.0 was just [recently released](https://github.com/tensorflow/tensorflow/releases/tag/v1.2.0). Why don't we stick with pip and can switch to a precompiled binary when it's up to date.